client/src/components/UploadAndDisplayImage.js

```diff
@@ -0,0 +1,31 @@
+
+
+import React, { useState } from "react";
+const UploadAndDisplayImage = () => {
+  const [selectedImage, setSelectedImage] = useState(null);
+
+  return (
+    <div>
+      {selectedImage && (
+        <div>
+        <img alt=" " width={"250px"} src={URL.createObjectURL(selectedImage)} />
+
+        <br />
+        <button onClick={()=>setSelectedImage(null)}>Change Profile</button>
+        </div>
+      )}
+      <br />
+      <br /> 
+      <input
+        type="file"
+        name="myImage"
+        onChange={(event) => {
+          console.log(event.target.files[0]);
+          setSelectedImage(event.target.files[0]);
```

Comment:
you should change this part...instead of just setting it to a hook, create a function that stores the image file on Mongo, so it won't be temp.
here is the link if you want to know how to store image file in Mongo https://www.geeksforgeeks.org/upload-and-retrieve-image-on-mongodb-using-mongoose/